src/HM6/Animal.java

```diff
@@ -0,0 +1,13 @@
+package HM6;
+
+abstract class Animal {
+    protected int _lenght;
+    protected double _height;
```

Comment:
имена на _ не приняты в Java
```suggestion
    protected int length;
    protected double height;
```